show/muxcable.py

```diff
@@ -412,3 +432,214 @@ def cableinfo(port):
 
     body = [[vendor, part_num]]
     click.echo(tabulate(body, headers=headers))
+
+
+@muxcable.group(cls=clicommon.AbbreviationGroup)
+def hwmode():
+    """start configuring cable in hardware mode on a port"""
```

Comment:
This is a `show` command, which does not configuration. Please rephrase this appropriately.

Reply:
fixed